src/textual/reactive.py

```diff
@@ -28,12 +26,6 @@
 ReactiveType = TypeVar("ReactiveType")
 
 
-class _NotSet:
-    pass
-
-
-_NOT_SET = _NotSet()
-
 T = TypeVar("T")
```

Comment:
actually, this slipped past me. Maybe this one too is unused?

Reply:
Looks like it